x-pack/plugins/event_log/generated/mappings.json

```diff
@@ -45,6 +45,10 @@
                 "outcome": {
```

Comment:
these are the two fields from the alert execution status, that weren't already populated somehow in the event, so added as new fields, in what I think are appropriate places - `reason` is a pre-existing. ECS field, but `status` is a new field in the custom ECS `kibana.alerting`field